LongWarOfTheChosen/Config/XComLW_Overhaul.ini

```
@@ -1647,6 +1647,11 @@ DEFAULT_MISSION_EXPERIENCE_WEIGHT=5.7f
 +CLASS_MISSION_EXPERIENCE_WEIGHTS=(SoldierClass="PsiOperative", MissionExperienceWeight=5.8f)
 +CLASS_MISSION_EXPERIENCE_WEIGHTS=(SoldierClass="Spark", MissionExperienceWeight=5.7f)
 
+; A list of soldier classes that will be excluded when mission XP is parceled
+; out at the end of missions
+; INELIGIBLE_FOR_MISSION_XP="Spark"
+; CLASSES_INELIGIBLE_FOR_MISSION_XP="Spark
```

You only need to put in the actual config variable (`INELIGIBLE_FOR_MISSION_XP` no longer exists) and this adds a missing double quote:
```suggestion
; CLASSES_INELIGIBLE_FOR_MISSION_XP="Spark"
```